rust/integration-tests/src/diff_tests/all_tests.rs

```diff
@@ -171,4 +212,55 @@ mod test {
             },
         ]
     }
+
+    // Helper function to generate output files for each table
```

Comment:
These helpers are already part of the SDK testing framework, we could reuse them rather than redefine here